pkg/jobs/jobclient.go

```diff
@@ -272,6 +275,9 @@ func (c *JobClient) LaunchK8sJob(repo result.Repository, execution testkube.Exec
 					return
 				}
 
+				// metrics increase
+				c.metrics.IncExecution(execution)
```

Comment:
Actually, worth to add it to LaunchK8sJobSync method as well. Methods need to be merged, treat it as a technical debt for now

Reply:
Sure, done